exercise-shopping-cart/models/product-model.js

```diff
@@ -16,6 +16,7 @@ class ProductModel extends Observable {
     let productQuantity = this.products[parseInt(index)].quantity;
```

Comment:
@trungpham-supreme You should move the parseInt out of the access array index. If the parserInt has an error, it will be hard to debug to find what happened.